model-optimizer/mo/utils/broadcasting.py

```diff
@@ -103,3 +122,19 @@ def bi_directional_broadcasting(input_value: np.array, second_shape: np.array):
         'The tensor of shape "{}" cannot be bi-directionally broadcasted to shape "{}"'.format(input_value.shape,
                                                                                                second_shape)
     return np.array(input_value * np.ones(second_shape), dtype=input_value.dtype)
+
+
+def explicit_broadcasting(input_value: np.array, target_shape: np.array, axes_mapping: np.array) -> np.array:
+    """
+    Explicit broadcasting of input tensor. Resulting shape is equal to target_shape except for axes specified in axes_mapping
+    :param input_value: input value to broadcast
+    :param target_shape: target shape
+    :param axes_mapping: a list of axis indices, each index maps an axis from the input_value to axis in the output
+    :return: broadcasted value
+    """
+    res_shape = explicit_shape_broadcasting(input_value.shape, target_shape, axes_mapping)
+    axes_mapping = map(lambda axis: axis + len(target_shape) if axis < 0 else axis, axes_mapping)
```

Comment:
lets have an assertion that all the values are within 0...rank-1 interval

Reply:
Done.